aws/resource_aws_lambda_function_test.go

```diff
@@ -1724,7 +1724,7 @@ resource "aws_lambda_function" "lambda_function_test" {
     function_name = "%s"
     role = "${aws_iam_role.iam_for_lambda.arn}"
     handler = "exports.example"
-    runtime = "nodejs6.10"
+    runtime = "nodejs8.10"
```

Comment:
The `TestAccAWSLambdaFunction_updateRuntime` test that references this test configuration should be performing an update of the `runtime` argument and this change will make the runtime value the same as the first test step. Can you please update this to another runtime value (e.g. `nodejs10.x`) and ensure the test check matches the new value? Thanks.

Reply:
Addressed: https://github.com/terraform-providers/terraform-provider-aws/pull/9341/commits/ccfd9575770d6aca8e255dea534d25c924f9d179